src/terra-dev-site/app/Application.1.app.mdx

```diff
@@ -0,0 +1,97 @@
+import { Badge } from 'terra-application/package.json?dev-site-package';
+
+<Badge />
+
+# Terra Application
+
+A React-based application framework from Cerner.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-application`
+
+## Why create an application framework?
+
+As we have developed web applications at Cerner we have found that there are many questions that must be answered at the outset of an application's development and that many of those question have a common answer that can be provided by an application framework.
+
+- How will this application consume existing components developed for other applications? How will interoperability be defined? How will dependencies be managed?
+- How will this application have a user experience that is consistent with that of other applications? How will that consistency be maintained over time?
```

Comment:
These points are supremely beneficial for our internal usage, but IMO they don't really have the same pull when we're taking about our open source offerings. I can imagine someone coming across this saying "I'm building one application, is this for me?"

We might be better off getting on our stump here and reiterating this framework's focus on responsive, accessible, globalized, complex single-page workflows?

Reply:
I have a hard time imagining developers choosing to use Terra unless they are interested in integrating with the Cerner application ecosystem. I think it's a valid point to reiterate the points you bring up as well, but I still think this point is relevant.

Reply:
I consolidated the sections.